src/index.html

```diff
@@ -1,14 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+              content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <img
+      src="images/logo.png"
+      alt="logo"
+      class="logo"
+      >
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="list__item">
+            <a
+             data-qa="hover"
+             class="nav__link"
+             href="#">Ноутбуки и Компьютеры
+            </a>
```

Comment:
перенось текст на новий рядок
```suggestion
            <a
             data-qa="hover"
             class="nav__link"
             href="#"
           >
               Ноутбуки и Компьютеры
            </a>
```